### DISCUSSION TITLE
Replicate tmux default tab title format

### DISCUSSION THREAD

**Discussion by hermitmaster** [kovidgoyal/kitty#5679] [Q&A] [answered]
I am trying to find a way to replicate the default tmux tab title formatting in kitty. i.e. `1: ~/.config - nvim`. Generically, `${index}: ${PWD} - ${CURRENT_ACTIVE_PROCESS}`. Any help is appreciated.

**Comment by kovidgoyal** [comment, marked as answer]
{index}: {tab.active_wd} - {title}


On Wed, Nov 16, 2022 at 02:13:07PM -0800, Dennis Rausch wrote:
> I am trying to find a way to replicate the default tmux tab title formatting in kitty. i.e. `1: ~/.config - nvim`. Generically, `${index}: ${PWD} - ${CURRENT_ACTIVE_PROCESS}`. Any help is appreciated.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5679
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by hermitmaster** [reply]
The only problem with that (I tried that already) is that it shows the PWD twice when sitting at a shell prompt.

**Comment by kovidgoyal** [comment]
So turn off title setting in shell integration
https://sw.kovidgoyal.net/kitty/shell-integration/#configuration

**Reply by hermitmaster** [reply]
I’ll give that a try. Thanks!

**Comment by hermitmaster** [comment]
@kovidgoyal thanks for your help. I ended up needing to do a couple of things. I had to setup `precmd` and `preexec` functions in zsh and set `shell_integration no-title`

See (.zshrc):
```sh
function precmd {
  print -Pn "\e]0;%~ - zsh\a"
}

function preexec {
  print -Pn "\e]0;%~ - $1\a"
}
```
